ios/Helio.xcodeproj/project.pbxproj

```diff
@@ -16,7 +16,7 @@
 		2D02E4BD1E0B4A84006451C7 /* Images.xcassets in Resources */ = {isa = PBXBuildFile; fileRef = 13B07FB51A68108700A75B9A /* Images.xcassets */; };
 		2D02E4BF1E0B4AB3006451C7 /* main.m in Sources */ = {isa = PBXBuildFile; fileRef = 13B07FB71A68108700A75B9A /* main.m */; };
 		2DCD954D1E0B4F2C00145EB5 /* BetterSocialTests.m in Sources */ = {isa = PBXBuildFile; fileRef = 00E356F21AD99517003FC87E /* BetterSocialTests.m */; };
-		2E7357E3EDC718B14778FD80 /* BuildFile in Frameworks */ = {isa = PBXBuildFile; };
+		2E7357E3EDC718B14778FD80 /* (null) in Frameworks */ = {isa = PBXBuildFile; };
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `(null) in Frameworks` reference could potentially cause issues. It seems like a file or framework that was supposed to be linked here is missing. Please ensure that all necessary files and frameworks are correctly linked.
```diff
- 		2E7357E3EDC718B14778FD80 /* (null) in Frameworks */ = {isa = PBXBuildFile; };
+ 		2E7357E3EDC718B14778FD80 /* [YourFramework] in Frameworks */ = {isa = PBXBuildFile; };
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->